lib/web/apiserver_test.go

```diff
@@ -4436,6 +4541,60 @@ func TestParseSSORequestParams(t *testing.T) {
 	}
 }
 
+func TestClusterDesktopsGet(t *testing.T) {
+	env := newWebPack(t, 1)
```

Comment:
This is kind of a slow test. Can we use `t.Parallel()` so it runs in parallel with others or is there some reason why the web tests can't be parallelized?